site/_docs/powered_by.md

```diff
@@ -137,3 +137,7 @@ It uses Calcite to transparently rewrite queries to use materialized views.
 is a SQL interface to a
 <a href="https://tinkerpop.apache.org/">Apache TinkerPop</a>-enabled
 graph database.
+
+### TBDS
+
+[TBDS](https://cloud.tencent.com/product/tbds) (Tencent Big Data Suite) is a reliable, safe, and easy-to-use big data processing platform based on Tencent's years of massive data processing experience. With the help of TBDS, you can choose a suitable big data analysis engine and corresponding real-time data development, offline data development, and algorithm development services to build your data warehouse in public, private, and non-cloud environments. Big data application services such as user portraits, precise recommendations, and risk management.
```

Comment:
The paragraph is too long. We'd better simplify it and focus on technique part.

Reply:
agree. see MaxCompute as example

Reply:
well let me change it